package.json

```diff
@@ -96,10 +98,19 @@
   "babel": {
     "presets": [
       "react",
-      "es2015"
+      ["env", {
+        "targets": {
+          "node": "current",
+          "browsers": "defaults"
+        }
+      }],
+      "flow"
     ],
     "plugins": [
-      "transform-flow-strip-types"
+      ["transform-runtime", {
+        "polyfill": false,
+        "regenerator": true
+      }]
```

Comment:
we only include "regenerator" to support async/await, but not the babel polyfill we likely never need for our targets.